syntax = "proto3";
import "timestamp.proto";

package yaroc;

message Punch { bytes raw = 1; }

message Punches {
  repeated Punch punches = 1;
  Timestamp sending_timestamp = 2;
}
